src/commcare_cloud/ansible/roles/commcarehq/templates/localsettings.py.j2

```diff
@@ -361,7 +361,7 @@ REPORT_CACHE = 'redis'
 
 
 redis_cache = {
-    'BACKEND': 'django_redis.cache.RedisCache',
+    'BACKEND': localsettings.get('REDIS_CACHE_CLASS', 'django_redis.cache.RedisCache'),
```

Comment:
this should be a template var with quotes: `'{{ localsettings.get('REDIS_CACHE_CLASS', 'django_redis.cache.RedisCache') }}'`